app/agent/workflows/mutex/LICENSE

```diff
@@ -0,0 +1,23 @@
+Temporal TypeScript Samples
```

Comment:
Another note in the readme?

Reply:
As noted here: https://github.com/hocus-dev/hocus/pull/63#discussion_r1175667868